src/app/components/Sidebar/Sidebar.jsx

```diff
@@ -54,17 +54,26 @@ const Sidebar = (props) => {
   const renderListSelector = data =>
     <ListSelector fieldsetProps={data} isJsEnabled={props.isJsEnabled} />;
 
+  const { displayDate = {}, displayAge } = props.displayInfo;
+  const booksfound = `${props.picksCount} Book${props.picksCount === 1 ? '' : 's'} Found`;
+
   return (
     <div className="sidebar nypl-column-one-quarter">
-      <nav aria-label="Breadcrumbs">
+      <nav aria-label="Breadcrumbs" className="book-filters-heading">
         <a href={config.recommendationsLink.url} className="back-link">
           <LeftWedgeIcon ariaHidden />
-          <span className="replaced-text visuallyHidden">Return to </span>
           {config.recommendationsLink.label}
         </a>
       </nav>
-      {renderListSelector(updateCurrentListSelectorValues(props))}
-      {renderBookFilters(props.isJsEnabled)}
+      <div className="book-filters-heading">
+        <h2 id="list-title" tabIndex="0">
```

Comment:
You can find this same component in `BookList` at https://github.com/NYPL/staff-picks/blob/development/src/app/components/BookList/BookList.jsx#L25.
I didn't refactor it into its own component but since you're working on it, you should do it.

Also, and very important, you cannot have the same `id` twice on the page and this renders it twice. It might be tricky to focus on one OR the other when switching between mobile/tablet/desktop.

Reply:
Sure. I thought you were working on it. Didn't want to do the same thing as we did on my last PR.